test/basicfuns.jl

```diff
@@ -44,6 +44,37 @@
     @test iszero(xlog1py(0 + im * 0, -1 + im * Inf))
 end
 
+@testset "xexpx" begin
+    @test iszero(xexpx(-Inf))
+    @test iszero(xexpx(0))
+    @test isnan(xexpx(NaN))
+
+    @test xexpx(1) ≈ exp(1)
+    @test xexpx(2) ≈ 2exp(2)
+
+    @inferred xexpx(0)
+    @inferred xexpx(0.)
+    @inferred xexpx(0f0)
+end
+
+@testset "xexpy" begin
+    @test iszero(xexpy(Inf, -Inf))
+
+    @test isnan(xexpy(NaN, -Inf))
+    @test isnan(xexpy(NaN, 1))
+    @test isnan(xexpy(0, NaN))
+
+    @inferred xexpy(0,-Inf)
+    @inferred xexpy(1,1)
+    @inferred xexpy(1.,-Inf)
```

Comment:
Again, check the actual types as well?

Reply:
Or, similarly, combine with value tests.